ui/components/multichain/pages/permissions-page/connection-list-item.js

```diff
@@ -94,12 +113,93 @@ export const ConnectionListItem = ({ connection, onClick }) => {
             >
               {t('connectedWith')}
             </Text>
-            <AvatarGroup
-              members={addressIconList}
-              limit={AVATAR_GROUP_LIMIT}
-              avatarType={AvatarType.ACCOUNT}
-              borderColor={BackgroundColor.backgroundDefault}
-            />
+            <Tooltip
+              position="bottom"
+              html={
+                <Box
```

Comment:
I think we should create a separate component ConnectedAccountsTooltip for this rather than rendering it inside this list item.